Dockerfile

```diff
@@ -17,11 +17,12 @@ RUN apk --no-cache update \
     && pip install --upgrade --no-cache-dir -r dev-requirements.txt \
     && apk del g++
 
-COPY ./syncserver /app/syncserver
-COPY ./setup.py /app
+COPY . /app
 RUN python ./setup.py develop
 
 # run as non priviledged user
 USER app
 
-ENTRYPOINT ["/usr/bin/dumb-init", "--"]
+# run the server by default
+ENTRYPOINT ["/usr/bin/dumb-init", "/app/docker-entrypoint.sh"]
```

Comment:
Nice to relocate some bash out of the yaml, also with `bin/ci/deploy-dockerhub.sh`. (Not sure how often the retry in the latter would ever trigger, but maybe someday will save a build.)